### ISSUE TITLE
End of life components

### ISSUE THREAD

**Issue by jimbro1000** [moogway82/XTjr#2]
ZTX449 is obsolete and unavailable (use PN2222?)
The 1.1A resettable fuse 16R110BPR is obsolete and in very limited supply

**Comment by moogway82** [comment]
Yup for the ZTX449 use a PN2222ATA - no idea why I used the ZTX449 now, it's been so long. The Xi8088 BOM specifies PN2222ATA and my circuit is based on that.
Again following the Xi8088 use the 512-PN2222ATA from Mouser, in-place of the 16R110BPR
As I based a lot of my design on the Xi8088 a lot of the components there are a good substitute for what I had used, I think my BOM is based on an older version.

**Comment by moogway82** [comment]
I've just had a peek at my board and I used a ZTX455 and I'm not sure why, it might've been picked a little randomly as I'm not very good at the differences between transisters, they probably looked similar and it seems to work tho. I'd stick with the PN2222ATA tho.
